pkg/crc/segment/segment.go

```diff
@@ -0,0 +1,74 @@
+package segment
+
+import (
+	"io/ioutil"
+	"os"
+
+	"github.com/code-ready/crc/pkg/crc/constants"
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/version"
+	"github.com/pborman/uuid"
+	"github.com/segmentio/analytics-go"
+)
+
+func NewSegmentClient() (analytics.Client, error) {
+	return analytics.NewWithConfig("cvpHsNcmGCJqVzf6YxrSnVlwFSAZaYtp", analytics.Config{
```

Comment:
I'm not sure what are the rules for segment write keys? Is this a personal key, something owned by crc, ...? Is this supposed to be secret?

Reply:
@cfergeau it is not suppose to secret, I did ask that before sending this PR